src/hooks/useAppState/useAppState.js

```diff
@@ -6,9 +6,9 @@ import getActions from './actions';
 import reducer, { initialState } from './reducer';
 import applyMiddleware from './middleware';
 
-export const IDB_KEY = 'items';
+export const IDB_KEY = 'thelist-state';
```

Comment:
Is anybody actually using the app? Because this change is not backwards compatible everybody will lose the items in their list. We can make it backwards compatible with a small amount of additional work

Reply:
Happy for this change to go ahead without the extra work, and I doubt anyone has been using it beyond us three 😂

Reply:
if we get a complaint we can still restore that data